src/index.html

```diff
@@ -7,6 +7,341 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header_logo">
```

Comment:
елементи створюємо через два нижніх підкреслювання
```suggestion
      <div class="header__logo">
```